src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/integration/SqsIntegrationTestBase.kt

```diff
@@ -71,12 +71,6 @@ class SqsIntegrationTestBase : TestBase() {
     scopes: List<String> = listOf()
   ): (HttpHeaders) -> Unit = jwtAuthHelper.setAuthorisation(user, roles, scopes)
 
-  @BeforeEach
-  fun cleanQueue() {
-    awsSqsVisitsMigrationClient.purgeQueue(PurgeQueueRequest(visitsMigrationQueueUrl))
```

Comment:
since this was only clearing down the visits queue I moved it into the visits int test instead